v20.1/backup.md

```diff
@@ -35,7 +35,8 @@ To view the contents of an enterprise backup created with the `BACKUP` statement
 
 ## Required privileges
 
-Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- `BACKUP` requires full write (including delete and overwrite) permissions to its target destination.
```

Comment:
Since we're mentioning the privileges on the bucket, I think we should say that: backup "requires full read and write ... ", since backup also needs to be able to read the bucket to see if there is an existing backup there.